app/views/record/_extended_info.html.erb

```diff
@@ -49,21 +49,19 @@
     </li>
   <% end %>
 
-  <% if @record.eds_isbn_print.present? %>
+  <% if @record.eds_isbns.present? %>
     <li>
-      ISBN: <%= @record.eds_isbn_print %>
+      ISBN: <% @record.eds_isbns.each do |isbn| %>
+        <%= isbn %> <br />
```

Comment:
Can you remove this hard line break and wrap the isbn in `<span class="isbn"></span>`